### ISSUE TITLE
Inconsistencies in z coordinate

### ISSUE THREAD

**Issue by Mrocza** [colobot/colobot#887]
After initiating a point variable without specifying its z coordinate: `point a(10,10)` Said coordinate will take a seemingly random value. In this case 0, 10,  ~1 or 1.4013e-045.

To reproduce: `while(true){ point a(10,10); message(a.z); wait(1);}`

0.1.9-alpha win64

**Comment by krzys-h** [comment]
Related: #872

**Comment by hexagonrecursion** [comment]
This bug appears to have been fixed in colobot-gold-0.2.0. `message(a.z)` now prints `0`

The bug was fixed by a66b3d06723a5e97655c65e178ceb2f05da1a233. Here is the relevant diff:

```diff
-    CBotVarNumberBase(const CBotToken &name) : CBotVarValue<T, type>(name) {}
+    CBotVarNumberBase(const CBotToken &name) : CBotVarValue<T, type>(name)
+    {
+        this->m_val = static_cast<T>(0);
+    }
```
